.github/workflows/gh-pages.yml

```diff
@@ -13,6 +15,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: "On engine release"
+        if: github.event.action == 'engine-release'
+        run: echo "Open Terms Archive engine release, version ${{ github.event.client_payload.version }}"
```

Comment:
What is the added value of this step? 🙂

Reply:
To know if the trigger of this action was a push on the main branch or the `engine-release` event

Reply:
Exactly :)